### ISSUE TITLE
Jacob & Esau hUd

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#2144]
the hUd for Jacob&Esau is super small in solo, prolly unintended ( side effect of co-op stuff )

**Comment by Lonrite** [comment]
yeah it's definitely unintended actually, you can't see the name of cards when ( by " when " I meant after ) you pick them up

**Comment by ZMBanshee** [comment]
Card names not appearing on pickup for J&E isn't new I'm 99% sure

**Comment by Lonrite** [comment]
> Card names not appearing on pickup for J&E isn't new I'm 99% sure

it's not


I just worded it poorly. not " when " you pick them up but " after " you pick up 


here's how it looks in Rep & Rep+ comparatively 


![wof Esau Rep](https://github.com/user-attachments/assets/ee1d49af-396c-4dd9-8b8d-19a63e6cbab1)
![wof Esau Rep+](https://github.com/user-attachments/assets/3c639325-db82-4de1-b440-df87d0116e02)



so unless you remember whether or no the card is reversed you kinda have to drop it to see unless you squeeze in your look real tight, which can be problematic in some situations 


to be fair it's clearly not reversed in the 2nd screenshot but if you're really rushing it can be problematic paying attention even to obvious stuff

**Comment by Lonrite** [comment]
also ig I should've posted all that stuff in my older general Rep+ oddities but ah well it's kinda w/e since it got slightly buried by other stuff anyways lol

& not all of the recent threads I made are specific to Rep+ anyways so it's w/e really

**Comment by ZMBanshee** [comment]
It seems kind of like Schoolbag's charge bar situation where your hud offset matters. if you use the hud offset of rebirth (max) then everything is visible, but if you have it more like the ab style (minimum) then some stuff goes off screen

**Comment by Lonrite** [comment]
> It seems kind of like Schoolbag's charge bar situation where your hud offset matters. if you use the hud offset of rebirth (max) then everything is visible, but if you have it more like the ab style (minimum) then some stuff goes off screen

yeah I use the minimum one, it's especially relevant post-patch because upper left part of the screen now has an effect similar to high hud offset. It was obviously made in mind with having your nickname above your Active Item & Hearts Indicator but in solo I'm not the biggest fan of it

it still doesn't bother me as much though as fortunes from pickups | consumables stacking on top of each other in an inconsistent-ish manner. it's oddly irritating at times
